tests/test_storage.py

```diff
@@ -47,6 +47,43 @@ def test_fetch_file(self) -> None:
                 test_string = open_dst_file.readline()
                 assert test_string == expected_string
 
```

Comment:
Since we are adding two new `put_file` and `put_object` functions to the `FileSystemDataStorage` class for local filesystems, let's add unit tests for them.